src/main/resources/l10n/JabRef_da.properties

```diff
@@ -2305,3 +2303,11 @@ Recommended_for_%0=
 This_might_be_caused_by_reaching_the_traffic_limitation_of_Google_Scholar_(see_'Help'_for_details).=
 
 Problem_downloading_from_%1=
+
+Manage_content_selectors=
+<field_name>=
```

Comment:
I think these strings should be localized without the brackets surrounding them.